hw/misc/hermes.c

```diff
@@ -399,18 +497,61 @@ static uint64_t hermes_bar0_read(void *opaque, hwaddr addr, unsigned size)
             val &= 0xFFFF;
             break;
         }
+    } else if (addr >= 0x1000 &&
```

Comment:
This is fine I think, but we need to fix the `if` above. `sizeof(struct hermes_bar0)` has increased, so it's no longer appropriate to use it in the check